.github/workflows/release.yml

```diff
@@ -72,6 +72,22 @@ jobs:
       - uses: actions/setup-java@v1.4.2
         with:
           java-version: 1.8
+      - name: Setup Github Package Registry
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          mkdir -p ~/.m2
+          cat <<EOF > ~/.m2/settings.xml
+          <settings>
```

Comment:
Is there any issue of escaping characters here?

Reply:
Not sure what you mean. This is just creating a file.

Reply:
like when I just copy your `cat <<EOF ...` statement to my terminal, it doesn't quite work, needs some kind of escaping or close quote or smth